daily-challenges/2024-09/17/haquyen.spec.ts

```
@@ -0,0 +1,88 @@
+function isPrime(num: number): boolean {
+  if (num <= 1) return false;
+  if (num <= 3) return true;
+
+  for (let i = 2; i * i <= num; i++) {
+    if (num % i === 0) return false;
+  }
+
+  return true;
+}
+
+export default isPrime;
```

Biome

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)